docs/articles/Getting-Started.md

```diff
@@ -0,0 +1,386 @@
+One App comes with a security standard in practice, progressive web app capability and
+many other features, its feature set can be configured by Holocron modules to your
+exact specification.
+
+This guide will get us started with One App and will go over some of the
+fundamental concepts of the framework. We will start from the beginning
+by creating our first Holocron module, then we will go through a few
+One App essentials to get up and running quickly.
+
+## 📖 Table of Contents
+* [Generating a Module](#generating-a-module)
+* [Running One App](#running-one-app)
+* [Adding CSS Styles](#adding-css-styles)
+* [Creating Routes](#creating-routes)
+* [Module State and Data](#module-state-and-data)
+* [Configuring One App](#configuring-one-app)
+* [Development](#development)
+
+### Generating a Module
+
+The first step to get started with One App is to generate a Holocron module.
+There are two different Holocron module types, root and child module. Let us
+start by creating a root Holocron module, the entry point to our micro frontend.
+
+```bash
+export NODE_ENV=development
+
+npx -p yo -p @americanexpress/generator-one-app-module \\
+-- yo @americanexpress/one-app-module
+```
+
+Once the command is executed, you will be prompted to fill out a few questions
+about your new module before it is generated. Starting off, we will want to name
+our module and make sure that it is a root module; feel free to answer as you
+wish for the other questions. Once the root module is generated we will be able
+to start developing with One App.
+
+> #### More Info
+>
+> [One App](https://github.com/americanexpress/one-app)
+>
+> ##### Packages
+>
+> [`react`](https://reactjs.org/)
+>
+> [`react-dom`](https://reactjs.org/docs/react-dom.html)
+>
+> [`holocron`](https://github.com/americanexpress/holocron/tree/main/packages/holocron)
+
+### Running One App
+
+Every Holocron module that is generated comes with pre-installed scripts that can
+be run afterwards. One of these scripts is the `npm start` script that starts up
+`one-app-runner`, which is our primary tool for local development.
+
+```bash
+npm start
+```
+
+When `one-app-runner` is fully loaded and running, we can navigate to `http://localhost:3000`
+and view our Holocron module in the browser. In another terminal window you can run
+
+```bash
+npm run watch:build
+```
+
+and this will watch for any changes made to your module, then update `one-app-runner`
+with the rebuilt module bundle. The command above uses `one-app-bundler` which can
+also be used to bundle our Holocron module.
+
+```bash
+npm run build
+```
+
+When ready to publish to production, make sure to set `NODE_ENV=production` before building.
+
+**Local Configuration**
+
+Inside the `package.json` of your Holocron module, you can
+include a `"one-amex": {}` property to configure some of the
+tools in our ecosystem.
+
+```json
+{
+  "name": "my-module",
+  "one-amex": {
+    "bundler": {
+      "webpackConfigPath": "webpack.config.js"
```

Comment:
Should we recommend  `webpackConfigPath` for the first setup

Reply:
Good point on that, I wanted readers to know that it is possible, however as you both mentioned, it is an advanced feature.